src/components/TermBox.vue

```diff
@@ -83,20 +88,24 @@ export default class TermBox extends ( Vue as VueConstructor<TermboxBindings> )
 	}
 
 	get hasAliases(): boolean {
-		return !( this.entityAliases( this.primaryLanguage ) == null );
+		return !( this.entityAliases( this.language.code ) == null );
```

Comment:
Rule 3 applies, but this method looks odd. Shouldn't it be
```suggestion
		return this.entityAliases( this.language.code ) !== null;
```